lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
This ensures that the footer doesn't print. The edX footer is really quite large, and it would be a waste of ink and paper to print that every time!